prow/jobs/jobs.yaml

```diff
@@ -7360,6 +7652,107 @@ presubmits:
           value: emrcontainers
         command: ["make", "test"]
 
+  aws-controllers-k8s/eventbridge-controller:
+  - name: eventbridge-kind-e2e
+    decorate: true
+    optional: false
+    always_run: true
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+      preset-test-config: "true"
+    extra_refs:
+    - org: aws-controllers-k8s
+      repo: code-generator
+      base_ref: main
+      workdir: false
+    - org: aws-controllers-k8s
+      repo: test-infra
+      base_ref: main
+      workdir: true
+    spec:
+      serviceAccountName: pre-submit-service-account
+      containers:
+      - image: public.ecr.aws/m5q3e4b2/prow:prow-integration-0.0.10
+        securityContext:
+          privileged: true
+        resources:
+          limits:
+            cpu: 8
```

Comment:
Just for my own education, is there a particular reasoning behind this choice of 8 VCPUs for EventBridge, or is this the standard resource limit we pick for e2e tests?

Reply:
@RedbackThomson Correct if i'm wrong:

i think pytest automatically detect the number of correct and the number of threads it can spin for e2e tests. This helps us in executing multiple tests at the same time.

Reply:
https://github.com/aws-controllers-k8s/test-infra/blob/main/scripts/pytest-local-runner.sh#L51